pkg/filters/containers.go

```diff
@@ -9,50 +9,24 @@ import (
 )
 
 type ContainerFilter struct {
-	Equal      []string
-	NotEqual   []string
-	Enabled    bool
-	bpfMapName string
+	*BPFStringFilter
 }
 
 func NewContainerFilter(mapName string) *ContainerFilter {
 	return &ContainerFilter{
-		Equal:      []string{},
-		NotEqual:   []string{},
-		Enabled:    false,
-		bpfMapName: mapName,
+		BPFStringFilter: NewBPFStringFilter(mapName),
 	}
 }
 
-func (filter *ContainerFilter) Parse(operatorAndValues string) error {
-	filter.Enabled = true
-
-	strFilter := &StringFilter{
-		Equal:    []string{},
-		NotEqual: []string{},
-	}
-
-	// Treat operatorAndValues as a string filter to avoid code duplication
-	err := strFilter.Parse(operatorAndValues)
-	if err != nil {
-		return err
-	}
-
-	filter.Equal = strFilter.Equal
-	filter.NotEqual = strFilter.NotEqual
-
-	return nil
-}
-
 func (filter *ContainerFilter) InitBPF(bpfModule *bpf.Module, conts *containers.Containers) error {
-	if !filter.Enabled {
+	if !filter.Enabled() {
```

Comment:
I don't see an Enabled() function of ContainerFilter, does this compile?

Reply:
It compiles, this struct embeds BPFStringFilter _in this commit_. Do note I change this filter in one of the last commits.

Reply:
fixed